cal-itp/benefits
# Feat: define an enrollment success template for agencies

Part of #1581 and #1571 

## How to test

1. Select MST
2. Confirm eligibility (any pathway)
3. Enter `/enrollment/success` into the URL bar
4. Confirm success message
5. Restart at 1. with SacRT
